executor/runtime/docker/docker_linux.go

```diff
@@ -59,6 +60,11 @@ const (
 )
 
 var systemServices = []serviceOpts{
+	{
+		unitName: "titus-container",
```

Comment:
Would this be better named, `titus-sidecar-container` ?

Or, is titus-executor PartOf titus-container?

Why do we need another unit though, why not just have things part of titus-executor?

Reply:
the target unit isn't a sidecar though. titus-container-sidecars@.target?